### PR TITLE
Remove unnecessary query param

### PR DESCRIPTION
**Pull request by seallard**
## Description
Closes https://github.com/Clinical-Genomics/streamline-delivery/issues/47.

Currently we have no use case where we want to exclude summaries, so I think it is worth to remove it for now to minimize confusion. Will need to remove in cigrid as well.

### Fixed
- Remove redundant query param

### This [version](https://semver.org/) is a
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: cg/services/orders/order_service/utils.py [hunk 1]
```diff
@@ -24,6 +24,9 @@ def create_orders_response(
 def _add_summaries(orders: list[Order], summaries: list[OrderSummary]) -> list[Order]:
     order_map = {order.id: order for order in orders}
     for summary in summaries:
-        order = order_map[summary.order_id]
-        order.summary = summary
+        try:
+            order = order_map[summary.order_id]
+            order.summary = summary
+        except KeyError:
+            continue
```
Comment (Vince-janv) [inline cg/services/orders/order_service/utils.py:31]: Which situation do the exceptions solve? Is there a missmatch between the number of orders and summaries?

File: cg/services/orders/order_service/utils.py [hunk 2]
```diff
@@ -24,6 +24,9 @@ def create_orders_response(
 def _add_summaries(orders: list[Order], summaries: list[OrderSummary]) -> list[Order]:
     order_map = {order.id: order for order in orders}
     for summary in summaries:
-        order = order_map[summary.order_id]
-        order.summary = summary
+        try:
+            order = order_map[summary.order_id]
+            order.summary = summary
```
Comment (islean) [inline cg/services/orders/order_service/utils.py:29]: When would a key error occur here?

**Review by Vince-janv — APPROVED** [review]
Yeah, with how quick the request to trailblazer was I think we can drop `include_summary`

**Review by islean — APPROVED** [review]
I like it! Why do we need to catch key error exceptions though? Does Trailblazer leave out fields with no entries?

File: cg/services/orders/order_service/utils.py (hunk 1 above)
Comment (seallard) [reply cg/services/orders/order_service/utils.py:31]: We can leave them out, but this one occurs if an analysis is tagged with an order that has been removed.

Comment (seallard) [reply cg/services/orders/order_service/utils.py:31]: Good point, this one can be skipped. There will always be a matching summary and order

File: cg/services/orders/order_service/utils.py (hunk 2 above)
Comment (seallard) [reply cg/services/orders/order_service/utils.py:29]: It won't, good point

**Comment by seallard** [comment]
> I like it! Why do we need to catch key error exceptions though? Does Trailblazer leave out fields with no entries?

Removed one check. The other check ensures the endpoint does not crash when trailblazer returns analyses which the corresponding order has been removed for. That should not occur in general, but I'd rather not cigrid breaks if it would happen at some point.

File: cg/services/orders/order_service/utils.py (hunk 1 above)
Comment (seallard) [reply cg/services/orders/order_service/utils.py]: This exception was hiding a bug from a function (`filter_orders_by_ids`) :+1:

**Comment by seallard** [comment]
> I like it! Why do we need to catch key error exceptions though? Does Trailblazer leave out fields with no entries?

True, removed both of them :+1: We always create summaries for existing orders and retrieve the corresponding analyses summaries, so there should never be any such exception

**Pull request merged**